src/plugins/opensearch_dashboards_react/public/code_editor/editor_theme.ts

```diff
@@ -117,7 +120,7 @@ export function createTheme(
       'editorWidget.border': standardizeColor(euiTheme.euiColorLightShade),
       'editorWidget.background': standardizeColor(euiTheme.euiColorLightestShade),
       'editorCursor.foreground': standardizeColor(euiTheme.euiColorDarkestShade),
-      'editorSuggestWidget.selectedBackground': standardizeColor(euiTheme.euiColorLightShade),
+      // 'editorSuggestWidget.selectedBackground': standardizeColor(euiTheme.euiColorLightShade),
```

Comment:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#do-not-comment-out-code are we sure we want to delete this?